s19/custom_mixin.rb

```diff
@@ -0,0 +1,42 @@
+module Purchaseable
+  def purchase(item)
+    "#{item} has been purchased!"
+  end
+end
+
+class Bookstore
+  include Purchaseable
+end
+
+class Supermarket
+  include Purchaseable
+end
+
+# Inherits from Supermarket
+class Petshop < Supermarket
+end
+
+x = Bookstore.new
+p x.purchase("Libro")
+y = Supermarket.new
+p y.purchase("Arroz")
+z = Petshop.new
+p z.purchase("Perro")
```

Comment:
Why can you call `purchase` on this instance of the `Petshop` class?

`z.purchase("Perro")`

Reply:
Because the < symbol assumes that Petshop inherits from Supermarket and this includes the Purchaseable module, then you can use the purchase method.